src/main/java/org/spongepowered/api/entity/Entity.java

```diff
@@ -246,4 +248,16 @@
      */
     void remove();
 
+    /**
+     * Damages this {@link Entity} with the given {@link Cause}. It is
+     * imperative that a {@link DamageSource} is included
+     * with the cause for maximum compatibility with plugins and the game
+     * itself.
+     *
+     * @param damage The damage to deal
+     * @param cause The cause of the damage
+     * @return True if damaging the entity was successful
+     */
+    boolean damage(double damage, Cause cause);
```

Comment:
I personally don't find it to be useful to store an extra variable in the manipulator for throwing an event since this is a function call to throw a new damage event.